libs/application/templates/party-letter/src/lib/answerValidators.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Application,
+  AnswerValidator,
+  AnswerValidationError,
+} from '@island.is/application/core'
+import isNumber from 'lodash/isNumber'
+import { UserCompany } from '../dataProviders/CurrentUserCompanies'
+
+const buildValidationError = (
```

Comment:
Could be moved inside the core to be reused across templates

Reply:
Nicked this from parental-leave, will move into core and update other templates using the same solution